pkg/runtime/inject/inject.go

```diff
@@ -14,6 +14,7 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
+// Package inject has been deprecated, and will be removed in v0.10.
```

Comment:
Use the golang `// Deprecated:` [convention](https://github.com/golang/go/wiki/Deprecated) directly
```suggestion
// Deprecated: Use manager.Options fields directly. This package will be removed in v0.10.
```